app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] As you have done in the PostsController, I'll kindly advise you to followed the same methodology here. To create a comment for a post, we will equally required two default Rails actions(`:new`, `:create`). This would equally permit you to construct comment form at app/views/comments/new.html.erb :+1: . Working like this is great in Rails. This ease debugging, extending your codes in future or working together in a team. I'll kindly advise you to refactor your codes in this manner :+1: